### ISSUE TITLE
{Linked:Bug3397279}[Adaptive Cards > Designer] - The menu item appears after expanding the 'Add an action' button is not accessible with keyboard.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8826]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/designer/
OS Version: 23H2 (OS Build 26040.1000)
Browser Version: Version 122.0.2365.3 (Official build) dev (64-bit)
Screen Reader: Narrator

**Repro Steps:**
1. Open the above URL on Edge Dev URL.
2. Press tab key and navigate to the 'New Card' and press enter key to activate it.
3. New Card dialog will be appeared.
4. Navigate to the 'Blank Card' using tab key and press enter key to activate it.
5. Navigate to the 'Add an Action' button and press enter key to activate it.
6. Press tab or arrow key and observe the issue.

**Actual Result:**
The menu item appears after expanding the 'Add an action' button is not accessible with keyboard, on pressing down arrow key focus does not move inside the menu items and after pressing tab key focus moves to the next control and the menu item remains expanded.

**Expected Result:** 
The menu item appears after expanding the 'Add an action' button should be accessible with keyboard up/down arrow keys and after pressing tab focus should land on next control and the menu item should collapse.

**Note:**
After adding any item from the add an action with mouse a clear button appears which is not accessible with keyboard

**User Experience:**
If menu items and clear button will not be accessible with keyboard, then the users dependent on the keyboard will be unable to access the UI properly and may face trouble.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/keyboard

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

![The menu item appears after expanding the 'Add an action' button is not accessible with keyboard](https://github.com/microsoft/AdaptiveCards/assets/95913748/c250b1bf-721c-4a94-a84d-616ed10d71b6)

https://github.com/microsoft/AdaptiveCards/assets/95913748/e434e30b-76a3-4ebd-87df-954b8c3fd559

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
Hi @anna-dingler,

Any update about this bug?

**Comment by anna-dingler** [comment]
Hi @vagpt,

I do not have any updates on this bug. At this time, my team doesn't have the bandwidth to resolve this issue ahead of schedule. It is currently budgeted for our April sprint.

Thanks,
Anna

**Comment by vagpt** [comment]
Hi @anna-dingler,

This issue is still repro at our end on the environment below, hence reactivating this issue.

Test Environment:
URL: https://adaptivecards.io/designer/
OS Version: 24H2 (OS Build 26090.1000)
Browser Version: Version 126.0.2566.1 (Official build) dev (64-bit)
Keyboard: Logitech K480

**Attachment:**


https://github.com/microsoft/AdaptiveCards/assets/95913748/5c4162d8-9e8f-4d2b-abe3-2c43ad7ed9ee

**Comment by anna-dingler** [comment]
Hi @vagpt, this fix is pending release on our site. I'll add the appropriate tag - apologies for the confusion.

**Comment by vagpt** [comment]
Hi @anna-dingler,

Hope you are doing great.

As this issue is still repro'ing at our end so could you please let us know by when the changes will be merged on the site.

**Comment by vagpt** [comment]
Hi @anna-dingler,

Hope you are doing good.

Any update on this issue as this is hitting the dependent team SLA?

**Comment by anna-dingler** [comment]
Hi @vagpt,

This issue has been resolved, and the fix has been released both on the website and in our JS package.

**Comment by vagpt** [comment]
This issue is working fine on the environment below, hence closing this issue.

**Test Environment:**
URL: https://adaptivecards.io/designer/
OS Version: 24H2 (OS Build 26244.5000)
Browser Version: Version 128.0.2690.3 (Official build) dev (64-bit)

**Attachment:**

Uploading Untitled video - Made with Clipchamp (44).mp4…
